### PR TITLE
PGB-47 Implement use of events in relate. Split relate job into multi…

### PR DESCRIPTION
**Pull request by JKoetsier**
…ple per-attribute jobs

### CODE REVIEW

File: src/gobupload/storage/update_table.py [hunk 1]
```diff
@@ -431,232 +418,68 @@ def _get_query(self):
...
-        return _execute(query)
-
-    def _write_delete_events(self, events):
-        """Updates database with deletes
-
-        :param events:
-        :return:
-        """
-        ids = ','.join([str(event['rel__gobid']) for event in events])
+        return relation
```
Comment (Vegter) [inline src/gobupload/storage/update_table.py]: return {
    field: datetime.combine(relation.get(field), datetime.min.time())
    for field in timestamps
    if relation.get(field) and isinstance(relation.get(field), date)
}

File: src/gobupload/storage/update_table.py [hunk 2]
```diff
@@ -87,35 +89,47 @@ def _select_expressions(self):
...
+        #     'src_volgnummer': '',
+        #     'derivation': '',
+        #     'dst_source': '',
+        #     'dst_id': '',
+        #     'dst_volgnummer': '',
+        #     'bronwaarde': '',
+        #     'begin_geldigheid': '',
+        #     'eind_geldigheid': '',
+        # }
+
```
Comment (Vegter) [inline src/gobupload/storage/update_table.py]: Why not put the example in the method comment. But also: what is the purpose of an empty example?

File: src/gobupload/storage/update_table.py (hunk 1 above)
Comment (JKoetsier) [reply src/gobupload/storage/update_table.py]: I knew it ;) Updated (spot the difference)

File: src/gobupload/storage/update_table.py (hunk 2 above)
Comment (JKoetsier) [reply src/gobupload/storage/update_table.py]: Removed it. Was more of a help during development for which columns to return, but this is easy to read in the code now.

**Pull request merged**